Player/src/main/java/dk/gruppeseks/bodtrd/player/PlayerPlugin.java

```diff
@@ -41,7 +41,10 @@ public void start(GameData gameData, Map<Integer, Entity> world, List<IEntityPro
         processors.add(_processor);
 
         _gameData = gameData;
-        gameData.setPlayerPosition(_entity.get(Position.class));
+        Body body = _entity.get(Body.class);
+        Position pos = _entity.get(Position.class);
+        _gameData.setPlayerPosition(pos);
```

Comment:
how about we calculate the centre of the player here and only pass the center position?